packages/nimble-components/src/anchor-menu-item/styles.ts

```diff
@@ -76,7 +76,7 @@ export const styles = css`
         height: ${iconSize};
     }
 
-    :host(.indent-1) .start {
+    :host(.indent-1) [part='start'] {
```

Comment:
All else equal I think I prefer the class syntax after seeing the diff 🤐

We haven't really leveraged Shadow Parts all that much and not really sure how much we will since our goal is to lean more towards opinionated instead of flexible and exposing parts of our controls to restyle externally. Pretty much all the [existing usages](https://dev.azure.com/ni/_search?action=contents&type=code&text=%22%3A%3Apart%22%20ext%3Ascss) seem questionable too. So I don't think I want to over encourage the use, just keep it as a semi-quiet escape hatch.

At the same time it's internal so if it doesn't feel worth the effort to switch the selectors over that's fine. It does look like every selector targetting `[part=` except one (action part of input) has a corresponding class of the same name. If we go the class route we could clean that up and just remove all the `[part=` selectors from our style selectors.

Reply:
I don't mind switching to use the class syntax. The thing that made me a bit hesitant to do that was that the classes only get applied when something is in the slot. I hadn't realized that until I started working on this PR, but I wasn't sure if that would break any styling assumptions.